spec/controllers/conferences_controller_spec.rb

```diff
@@ -26,6 +26,13 @@
     end
   end
 
+  describe 'GET #current' do
+    it 'redirects to first conference splashpage' do
+      current = Conference.first
+      expect(response).to redirect_to(conference_path(current.short_title))
```

Comment:
You are not invoking an action in the controller. The thing this test is testing, when is it supposed to happen?